website/common/locales/en/tasks.json

```diff
@@ -27,15 +20,11 @@
   "save": "Save",
   "addChecklist": "Add Checklist",
   "checklist": "Checklist",
-  "checklistText": "Break a task into smaller pieces! Checklists increase the Experience and Gold gained from a To Do, and reduce the damage caused by a Daily.",
   "newChecklistItem": "New checklist item",
-  "expandChecklist": "Expand Checklist",
```

Comment:
this one is used here https://github.com/HabitRPG/habitica/blob/develop/website/client/src/components/tasks/task.vue#L184

Reply:
This is a good example, thanks. I believe I have understood the issue. I will correct the strings you mentioned and watch out for such cases in the next searches.